plutus-tutorial/doc/plutus-tx.adoc

```diff
@@ -51,10 +51,10 @@ The Plutus Tx compiler compiles Haskell _expressions_ (not values!), so
 naturally it takes a quote (representing an expression) as an argument. The
 result is a new quote, this time for a Haskell program
 that represents the _compiled_ program. In Haskell, the type of  `compile` is
-`TExpQ a -> TExpQ (CompiledCode a)`. This is just what we already said:
+`TExpQ a -> TExpQ (CompiledCode PLC.DefaultUni a)`. This is just what we already said:
 
 - `TExpQ a` is a quoted representing a program of type `a`.
-- `TExprQ (CompiledCode a)` is quote representing a compiled Plutus Core
+- `TExprQ (CompiledCode PLC.DefaultUni a)` is quote representing a compiled Plutus Core
 program.
 
 NOTE: `CompiledCode` has a type parameter `a` as well, which corresponds to the
```

Comment:
I added a sentence, but I think the actual solution would be to have

```haskell
data CompiledCodeIn uni a = ...
type CompiledCode = CompiledCodeIn PLC.DefaultUni
```

so that universes stuff does not leak to the user. But let's leave that for future.